src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceAssessmentPreview.vue

```diff
@@ -1,198 +1,192 @@
-          <p class="small">Child guidance</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.childGuidance }}</p>
-          <p v-if="assessment.childGuidanceReason" class="small">{{ assessment.childGuidanceReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Health, safety and nutrition</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.healthSafetyAndNutrition }}</p>
-          <p v-if="assessment.healthSafetyAndNutritionReason" class="small">{{ assessment.healthSafetyAndNutritionReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Developing an early childhood education curriculum</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.developAnEceCurriculum }}</p>
-          <p v-if="assessment.developAnEceCurriculumReason" class="small">{{ assessment.developAnEceCurriculumReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Implementing an early childhood education curriculum</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.implementAnEceCurriculum }}</p>
-          <p v-if="assessment.implementAnEceCurriculumReason" class="small">{{ assessment.implementAnEceCurriculumReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Fostering positive relationships with children under their care</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.fosteringPositiveRelationChild }}</p>
-          <p v-if="assessment.fosteringPositiveRelationChildReason" class="small">{{ assessment.fosteringPositiveRelationChildReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Fostering positive relationships with the families of children</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.fosteringPositiveRelationFamily }}</p>
-          <p v-if="assessment.fosteringPositiveRelationFamilyReason" class="small">{{ assessment.fosteringPositiveRelationFamilyReason }}</p>
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col cols="4">
-          <p class="small">Fostering positive relationships with co-workers</p>
-        </v-col>
-        <v-col cols="8">
-          <p class="small font-weight-bold">{{ assessment.fosteringPositiveRelationCoworker }}</p>
-          <p v-if="assessment.fosteringPositiveRelationCoworkerReason" class="small">
-            {{ assessment.fosteringPositiveRelationCoworkerReason }}
-          </p>
-        </v-col>
-      </v-row>
-    </template>
-  </ReferencePreviewCard>
+  <div>
```

Comment:
going to push back on this one. I rushed this one out before I left on vacation. 

This change won't work because this form is used for references. 

A new review form should be created specific for icra-five-year. The naming is a bit messed up. 

I'll fix it. 

Also just a heads up that our pattern is a bit different here
We separate the review form into sections. So I would recommend keeping to the pattern so it's consistent throughout our application.